### PR TITLE
Analytics Page PHP client v2 compatibility 

### PR DESCRIPTION
**Pull request by bsuravech**
Updates to the Analytics Overview for PHP client v2 compatibility.

### CODE REVIEW

File: view/adminhtml/layout/algolia_algoliasearch_analytics_index.xml [hunk 1]
```diff
@@ -14,14 +14,14 @@
                     <argument name="default_selection_name" xsi:type="string" translate="true">Default Store View</argument>
                 </arguments>
             </block>
-            <block class="Magento\Backend\Block\Template" name="analytics.overview.form" template="Algolia_AlgoliaSearch::analytics/form.phtml">
+            <block name="analytics.overview.form" template="Algolia_AlgoliaSearch::analytics/form.phtml">
```
Comment (vmalyk) [inline view/adminhtml/layout/algolia_algoliasearch_analytics_index.xml]: @bsuravech ,  i suggest to restore class param and prevent issue with using block methods in template( escapeHtml, escapeJs, etc)

File: view/adminhtml/layout/default.xml [hunk 2]
```diff
@@ -5,6 +5,10 @@
     </head>
 
     <referenceBlock name="before.body.end">
-        <block class="Magento\Backend\Block\Template" name="algolia_tracking" template="Algolia_AlgoliaSearch::tracking.phtml" />
+        <block class="Magento\Backend\Block\Template" name="algolia_tracking" template="Algolia_AlgoliaSearch::tracking.phtml">
```
Comment (vmalyk) [inline view/adminhtml/layout/default.xml]: Based  on name strategy for consent naming - algolia.tracking?

File: view/adminhtml/templates/tracking.phtml [hunk 3]
```diff
@@ -1,4 +1,11 @@
+<?php
+/** @var \Algolia\AlgoliaSearch\ViewModel\Adminhtml\Common $viewModel */
+$view = $this->getViewModel();
+$applicationId = $view->getApplicationId();
+?>
+
 <script>
+var applicationId = <?php echo json_encode($applicationId); ?>;
```
Comment (vmalyk) [inline view/adminhtml/templates/tracking.phtml]: based on Magento documentation and XSS prevention for single string (not object) Magento recommends to use escapeJS (ref. https://devdocs.magento.com/guides/v2.3/extension-dev-guide/xss-protection.html)

File: view/adminhtml/layout/algolia_algoliasearch_analytics_index.xml (hunk 1 above)
Comment (bsuravech) [reply view/adminhtml/layout/algolia_algoliasearch_analytics_index.xml]: Ah right on. Good catch @vmalyk!

File: view/adminhtml/templates/tracking.phtml [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+/** @var  \Magento\Backend\Block\Template $block */
 /** @var \Algolia\AlgoliaSearch\ViewModel\Adminhtml\Common $viewModel */
-$view = $this->getViewModel();
+$view = $block->getViewModel();
 $applicationId = $view->getApplicationId();
 ?>
 
 <script>
-var applicationId = <?php echo json_encode($applicationId); ?>;
+var algoliaAppId = '<?php echo $block->escapeHtmlAttr($applicationId) ?>';
```
Comment (vmalyk) [inline view/adminhtml/templates/tracking.phtml]: @bsuravech, right escape function is $block->escapeJs($applicationId), escapeHtmlAttr uses for escape value in html attrbutes

Comment (bsuravech) [reply view/adminhtml/templates/tracking.phtml]: ah word! thanks!

**Pull request merged**